docs/modules/ROOT/pages/explanation/decisions/night-maintenance.adoc

```diff
@@ -0,0 +1,35 @@
+= Offering Nighttime Maintenance
+
+== Problem
+
+Maintenance of APPUiO Cloud is currently done during daytime hours.
+Although we encourage and support our customers to deploy their applications in a highly available configuration,
+this has led to significant dissatisfaction among some customers.
+
+Many have expressed a preference for potentially disruptive node reboots to occur after office hours.
+This presents a challenge as we have applications on APPUiO Cloud that are dependencies for other clusters maintenance,
+necessitating the maintenance schedule to remain primarily as is to avoid impacting these interdependent systems.
```

Comment:
```suggestion
On the other hand, since a lot of maintenance infrastructure is hosted on APPUiO Cloud, we can't easily move the APPUiO Cloud maintenance into the same window as most production clusters.
Finally, we don't want to introduce additional outside office hours maintenance windows, to keep the number of evenings that our on-call engineers may need to handle maintenance issues as low as possible.
```